### ISSUE TITLE
Why not have <title></title> in the meta.html?

### ISSUE THREAD

**Issue by DmitryFillo** [nephila/django-meta#4]
I use meta_extended.html which looks like this:
```    
{% include "meta.html" %}
{% if meta.title %}<title>{{ meta.title }}</title>{% endif %}
```
I suggest add this to the django-meta: title tag is meta/seo tag too. For example, we can add property to the view mixin which enables/disables showing <title></title> in this way:
```
class MyView(MetadataMixin, View):
    title = 'Some page'
    title_tag = True # enables <title></title> in the meta.html
    description = 'This is an awesome page'
    image = 'img/some_page_thumb.gif'
    url = 'some/page/'
```

**Comment by yakky** [comment]
I agree that title it's one of the most important meta/seo tag; because of this I'd rather keep it rather shallow in the templates inheritance to make it easy to override it even if a very basic use of django-meta is made.
Keeping it into the meta.html template would make it almost impossibile to override at the template level, which can be annoying a times.
I'm anyway open to suggestion on how to implement this in a backward compatible way

**Comment by DmitryFillo** [comment]
Yep, but ``title_tag`` property will control this. ``True`` value means that you don't need overriding title (using only dynamic title generation for whole site) and so title tag will be in meta.html. ``False`` is default value and django-meta behaves as it is now. There are many cases that no need title overriding in the templates. For example, ajax sites with html5 history router.

**Comment by yakky** [comment]
I like the idea, sorry if I missed that.

**Comment by yakky** [comment]
@DmitryFillo Fancy contributing the feature? :)

**Comment by DmitryFillo** [comment]
@yakky I think I made the pull request in a few days)

**Comment by yakky** [comment]
Fixed by #10 
Thanks for your contribution!
